sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -13,10 +13,12 @@ namespace Azure.Core.Pipeline
     /// </summary>
     public static class HttpPipelineBuilder
     {
+        private static int DefaultPolicyCount = 8;
+
         /// <summary>
-        /// Creates an instance of <see cref="HttpPipeline"/> populated with default policies, customer provided policies from <paramref name="options"/> and client provided per call policies.
+        /// Creates an instance of <see cref="HttpPipeline"/> populated with default policies, user provided policies from <paramref name="options"/> and client provided per call policies.
         /// </summary>
-        /// <param name="options">The customer provided client options object.</param>
+        /// <param name="options">The user provided client options object.</param>
```

Comment:
```suggestion
        /// <param name="options">The user-provided client options object.</param>
```

nit: There's a bunch of these.

Reply:
Will fix